codedeploy/be/scripts/start.sh

```
@@ -0,0 +1,10 @@
+#!/bin/bash
+# scripts/start.sh
+
+APP_DIR="/home/ubuntu/app/"
+JAR_FILE="ahmadda-server-prod.jar"
+LOG_FILE="/home/ubuntu/app/app.log"
```

Shellcheck (0.10.0)

[warning] LOG_FILE appears unused. Verify use (or export if used externally).

(SC2034)